roles/auth/tasks/main.yml

```diff
@@ -90,6 +90,7 @@
       PUSHBOX_ENABLED: "{{ auth_pushbox_enabled }}"
       PUSHBOX_URL: "{{ auth_pushbox_url }}"
       PUSHBOX_KEY: "{{ auth_pushbox_key }}"
+      RECOVERY_CODE_COUNT: "{{ auth_recovery_code_count | default('8')}}"
```

Comment:
do you need the `default 8` here, it might use stuff from `roles/auth/defaults/main.yml`
cc @jrgm ?

Reply:
Removed!